CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 Changelog
 ---------
+* 0.40: *gluon 2016.2.x*
```

Comment:
This in fact *gluon v2016.2.7-3-g35c9f9ae*. See the commit 46bf0f6ff0ea477698307c604d89fd27a904a055 with the version bump. Maybe be a little more specific in this changelog?

Reply:
Danke für den Hinweis, habe ich angepasst.